.github/workflows/build.yml

```diff
@@ -0,0 +1,82 @@
+---
+# This workflow will combine the experiment CRs for different 
+# Categories of experiments into the experiment.yaml in respective
+# Chart directory. It will only push the changes from charts directory.
+
+name: ChaosCharts
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+
+      # Install golang
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '^1.13.1'
+
+      # Setup gopath
+      - name: Setting up GOPATH 
+        run: |
+          echo "GOPATH=${GITHUB_WORKSPACE}/go" >> $GITHUB_ENV
+          
+      # Checkout to the latest commit
+      # On specific directory/path
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{steps.getcommit.outputs.sha}}
+          path: go/src/github.com/${{github.repository}}    
+          
+      # Combine the experiment CRs
+      - name: Combine experiment CR
+        run: |
+          export PATH=$PATH:$(go env GOPATH)/bin
+          cd ${GOPATH}/src/github.com/${{github.repository}}
+          make combineExpCR
+        shell: bash
+
+      # Upload chart directory as an artifact
+      - name: Upload artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: ChaosExp-CR
+          path: |
+            go/src/github.com/${{github.repository}}/charts
+
+  push:
+    runs-on: ubuntu-latest
+    needs: build
+    if: github.event_name == 'push' && github.repository == 'litmuschaos/chaos-charts'
```

Comment:
Any specific reason for the github.repository

Reply:
Yes if we don't mention `github.repository` then while raising PRs we will be pushing in our forked repo and github actions will run this `push` job also as it is a push event for forked repo and  PR for the main repo.  For eg you can see the second commit on this PR was due to that only. 

We can also create separate files for each of them (build and push) with this, we can avoid such issues, and then the skipped job will also not display in the checks for every PR.

Having said that the current thing should also work fine. @ksatchit

Reply:
Ok. If there is not image build as an extra step should we separate out the workflows

Reply:
Have updated with different files PTAL. But we need this condition in push.yml so that it should not run for the forked repo.